web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -130,8 +130,12 @@
     }
   });
 
-  const onCopyShortcut = () => {
-    if (window.getSelection()?.type === 'Range') {
+  const onCopyShortcut = async () => {
+    const selection = window.getSelection();
+    if (selection?.type === 'Range') {
+      if (selection.toString().length > 0) {
+        await copyToClipboard(selection.toString(), true);
```

Comment:
Alternatively we can set `preventDefault = false` on the shortcut ([see here](https://github.com/immich-app/immich/blob/57d94bce6865d222ed56078224076a682017869e/web/src/lib/actions/shortcut.ts#L63-L65)), just return from this `if type == range`, and then call preventDefault after the if block.

Reply:
(thus letting the browser just go to the default copy behaviour)

Reply:
That is indeed better, didn't realize that `preventDefault` was true by default.